xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/TemplateProviderSheet.xml

```diff
@@ -42,56 +42,52 @@
   <content>{{velocity}}
 #set ($discard = $xwiki.jsx.use('XWiki.TemplateProviderSheet'))
 #set ($providerObj = $doc.getObject('XWiki.TemplateProviderClass'))
-#if ($editor == 'inline')
-  ; {{translation key="xe.templateprovider.name"/}}
-  : {{html}}&lt;input type="text" value="$!{doc.getTitle()}" name="title" /&gt;{{/html}}
-#end
-; {{translation key="xe.templateprovider.templatename"/}}
-#if ($editor == 'inline')
+{{html wiki="true"}}
```

Comment:
@Enygma2002 I don't think it's a good idea to use wiki syntax to create UIs or forms because it wasn't meant for that. I did it in the past but I stopped. Wiki syntax is meant for content, when you don't care that much about the final HTML structure or the applied styles. On the contrary, when you write UIs/forms you want 100% control of the final HTML structure in order to enhance it with JavaScript and style it with CSS. You don't want automatically generated paragraphs or spans, and you want to control the attributes of any element (including list items).